flask/app/api/parsers.py

```diff
@@ -2,6 +2,8 @@
 from datetime import datetime
 
 def datetype(x):
+    if not x:
```

Comment:
had to deal with empty string end dates with the caregivers. alternate solution would have been to change empty strings to null in the json sent from the frontend